development/attributions-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+yarn attributions:generate
+
+ATTRIBUTIONS_FILE="./attribution.txt"
+
+# check to see if there's changes on the attributions file
+if ! git diff --exit-code "$ATTRIBUTIONS_FILE"; then
+  echo "attribution.txt has changed!"
+  echo "run \`yarn attributions:generate\` and commit changes."
```

Comment:
I feel like this message is a bit misleading. It sounds like the user has changed `attribution.txt`, when in fact what has happened is that this file needs to be changed.

Perhaps this should say:

``` suggestion
  echo "The set of attributions kept in \`attribution.txt\` are out of date."
  echo "Run \`yarn attributions:generate\` to regenerate them and commit and push the changes."
  echo "If you're looking at a pull request, you can also post the comment \`@metamaskbot regenerate-attributions\` and the file will be automatically regenerated for you."
```